code/game/gamemodes/vampire/vampire_bite.dm

```diff
@@ -0,0 +1,20 @@
+/mob/living/carbon/human/proc/handle_vamp_biting(var/mob/living/carbon/human/M)
+	if(!is_vampire(M) || M == src || M.zone_selected != "head")
+		return FALSE
+	var/datum/antagonist/vampire/V = M.mind.has_antag_datum(ANTAG_DATUM_VAMPIRE)
+	if((NOBLOOD in dna.species.species_traits) || dna.species.exotic_blood || !blood_volume)
+		to_chat(M, "<span class='warning'>They have no blood!</span>")
+		return FALSE
+	if(is_vampire(src))
+		to_chat(M, "<span class='warning'>Your fangs fail to pierce [name]'s cold flesh</span>")
+		return FALSE
+	if(dna.species.name == "skeleton")
```

Comment:
You just checked for no blood species like 5 lines up

Reply:
W E W

Reply:
~This one is excusable since he is setting a different text~ nevermind you would have to put this BEFORE the "no blood species" otherwise it would go first
Also skeletons do have blood